client.go

```diff
@@ -29,14 +28,325 @@ func main() {
+	selectAllCapitals(sqlService)
+	selectCapitalNames(sqlService)
+}
+
+func createMappingForCapitals(sqlService sql.Service) {
+	// See: https://docs.hazelcast.com/hazelcast/5.1/sql/mapping-to-maps
+	fmt.Println("Creating a mapping...")
+	mappingQuery := fmt.Sprintf(`
+        CREATE OR REPLACE MAPPING capitals TYPE IMap
+		OPTIONS (
+			'keyFormat' = 'varchar',
+			'valueFormat' = 'varchar'
+		)
+	`)
+	ignored, err := sqlService.Execute(context.Background(), mappingQuery)
+	if err != nil {
+		panic(err)
+	}
+	defer ignored.Close()
+	fmt.Println("The mapping has been created successfully.")
+	fmt.Println("--------------------")
+}
+
+func clearCapitals(sqlService sql.Service) {
+	fmt.Println("Deleting data via SQL...")
+	ignored, err := sqlService.Execute(context.Background(), "DELETE FROM capitals")
+	if err != nil {
+		panic(err)
+	}
+	defer ignored.Close()
+	fmt.Println("The data has been deleted successfully.")
+	fmt.Println("--------------------")
+}
+
+func populateCapitals(sqlService sql.Service) {
+	fmt.Println("Inserting data via SQL...")
+	insertQuery := fmt.Sprintf(`
+        INSERT INTO capitals VALUES
+			('Australia','Canberra'),
+            ('Croatia','Zagreb'),
+			('Czech Republic','Prague'),
+			('England','London'),
+			('Turkey','Ankara'),
+			('United States','Washington, DC');
+	`)
+	ignored, err := sqlService.Execute(context.Background(), insertQuery)
+	if err != nil {
+		panic(err)
+	}
+	defer ignored.Close()
+	fmt.Println("The data has been inserted successfully.")
+	fmt.Println("--------------------")
+}
+
+func selectAllCapitals(sqlService sql.Service) {
+	fmt.Println("Retrieving all the data via SQL...")
+	ctx := context.Background()
+	result, err := sqlService.Execute(ctx, "SELECT * FROM capitals")
+	if err != nil {
+		fmt.Errorf("Querying: %w", err)
+	}
+	iterator, err := result.Iterator()
+	if err != nil {
+		fmt.Errorf("Acquaring iterator: %w", err)
+	}
+	for iterator.HasNext() {
+		row, err := iterator.Next()
+		if err != nil {
+			panic(err)
+		}
+		country, err := row.Get(0)
+		city, err := row.Get(1)
+		fmt.Printf("%s - %s\n", country, city)
+	}
+	fmt.Println("--------------------")
+}
+
+func selectCapitalNames(sqlService sql.Service) {
+	ctx := context.Background()
+	fmt.Println("Retrieving the capital name via SQL...")
+	result, err := sqlService.Execute(ctx, "SELECT __key, this FROM capitals WHERE __key = ?", "United States")
+	iter, err := result.Iterator()
+	if err != nil {
+		fmt.Errorf("acquaring iterator: %w", err)
+	}
+	for iter.HasNext() {
+		row, err := iter.Next()
+		if err == nil {
+			fmt.Errorf("iterating: %w", err)
+		}
+		country, err := row.GetByColumnName("__key")
+		city, err := row.GetByColumnName("this")
+		fmt.Printf("%s - %s\n", country, city)
+	}
+	fmt.Println("--------------------")
+}
+
+// jsonSerializationExample shows how to work with Hazelcast SQL queries via Maps that
+// contains JSON serialized values.
+func jsonSerializationExample(client *hazelcast.Client) {
```

Comment:
Content of this method is described step by step on the java sample. Can you also add it as a command? I'm talking about the following part
```
<ul>
     *     <li>Select single json element data from a Map</li>
     *     <li>Select data from Map with filtering</li>
     *     <li>Join data from two Maps and select json elements</li>
     * </ul>
```

Reply:
The only way to do that ATM is having a code block, something like:
```
// some documentation here
//    * Select single json element data from a Map.
//    * Select data from Map with filtering.
//    * Join data from two Maps and select json elements.
```
Alternatively, we can use something like the following (We've used that in the Go client, IIRC):
```
// (1) Select single json element data from a Map.
// (2) Select data from Map with filtering.
// (3) Join data from two Maps and select json elements.